assets/js/quiz.js

```diff
@@ -0,0 +1,70 @@
+import { questions } from './utils/questions.js';
+
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_refinement):** Consider making the quiz timer's duration configurable.

Hardcoding the timer's duration limits flexibility. It might be beneficial to allow it to be set through a function parameter or a configuration object.